app/src/main/res/xml/prefs.xml

```diff
@@ -9,50 +9,50 @@
         android:title="@string/general_settings">
 
         <CheckBoxPreference
-            android:defaultValue="@bool/preferences_auto_update_enabled_default_value"
-            android:key="auto_update"
-            android:title="@string/pref_auto_download" />
+            android:defaultValue="@bool/preference_default_value_auto_update_enabled"
```

Comment:
Personal opinion: Using string resources for key names and values makes looking up things very annoying. Not all duplication is bad.

Reply:
This was actually one of the top reasons to refactor this. The former distributed keys caused problems over the years.